rpcs3/Emu/RSX/Program/ProgramStateCache.cpp

```diff
@@ -587,11 +587,10 @@ bool fragment_program_compare::operator()(const RSXFragmentProgram& binary1, con
 		instIndex++;
 		// Skip constants
 		if (fragment_program_utils::is_constant(inst1._u32[1]) ||
-			fragment_program_utils::is_constant(inst1._u32[2]) ||
-			fragment_program_utils::is_constant(inst1._u32[3]))
+			fragment_program_utils::is_pair_constant(inst1._u64[1]))
```

Comment:
You can simplify it further by ORing.
```suggestion
			if (fragment_program_utils::is_pair_constant(inst1._u64[1] | inst1._u32[1])))
```

Reply:
Seems clang at least is still emitting 3 branches, even with all the bools ORed.